src/main/resources/application.properties

```diff
@@ -1,7 +1,8 @@
 spring.jpa.hibernate.ddl-auto=update
```

Comment:
I love the continual fight over this file

Reply:
LOL yes. After seeing this, I realized I just did that in my branch...